static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.js

```diff
@@ -74,9 +74,20 @@ const show = (): void => {
     msg.show(makeHtml(template, bindableClassNames));
 };
 
+const test = {
```

Comment:
I'm guessing `test` is exposed to allow the tests to call `onAgree` and `binableClassNames`. There's a common pattern used throughout our js where we export a method named `_` to denote that it's for testing purposes.

eg. `export const _ = { onAgree,  binableClassNames };`.

Reply:
ah alright!! i thought it was _test but the linter wouldn't let me name it like that so i went with test